data/lua/wml/find_path.lua

```diff
@@ -59,9 +59,8 @@ function wesnoth.wml_actions.find_path(cfg)
 				ignore_visibility = ignore_visibility
 			} )
 
-			if #path == 0 or cost >= 42424241 then
-				-- it's not a reachable hex. 42424242 is the high value returned for unwalkable or busy terrains
-			else
+			-- it's not a reachable hex. 42424242 is the high value returned for unwalkable or busy terrains
+			if #path ~= 0 and cost < 42424241 then
```

Comment:
This renders the comment incorrect. It was correct only because it wasn't in the `if` branch – it's now describing the situation when the condition is false, but in a way that makes it sound like it's describing the situation when the condition is true.

Reply:
Updated.

Reply:
What did you change…?